pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -56,6 +56,12 @@ elif command -v pacman >/dev/null 2>&1; then
 			pacman -Syu --noconfirm sshfs
 		fi
 	fi
+elif command -v zypper >/dev/null 2>&1; then
+	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ]; then
+		if ! command -v sshfs >/dev/null 2>&1; then
+			zypper install -y sshfs
+		fi
+	fi
```

Comment:
```
suda@lima-opensuse:~> containerd-rootless-setuptool.sh nsenter containerd-fuse-overlayfs-grpc /run/user/501/containerd-fuse-overlayfs.sock /home/suda.linux/.local/share/containerd-fuse-overlayfs 
INFO[0000] containerd-fuse-overlayfs-grpc Version="v1.0.2" Revision="0c586ae77b866c333af73c6e9f9bf7a4b64cb06b" 
error: fuse-overlayfs not functional, make sure running with kernel >= 4.18: failed to mount fuse-overlayfs ({Type:fuse3.fuse-overlayfs Source:overlay Options:[lowerdir=/home/suda.linux/.local/share/containerd-fuse-overlayfs/fuseoverlayfs-check824306133/lower2:/home/suda.linux/.local/share/containerd-fuse-overlayfs/fuseoverlayfs-check824306133/lower1]}) on /home/suda.linux/.local/share/containerd-fuse-overlayfs/fuseoverlayfs-check824306133/merged: mount helper [mount.fuse3 [overlay /home/suda.linux/.local/share/containerd-fuse-overlayfs/fuseoverlayfs-check824306133/merged -o lowerdir=/home/suda.linux/.local/share/containerd-fuse-overlayfs/fuseoverlayfs-check824306133/lower2:/home/suda.linux/.local/share/containerd-fuse-overlayfs/fuseoverlayfs-check824306133/lower1 -t fuse-overlayfs]] failed: "": exec: "mount.fuse3": executable file not found in $PATH
```

Reply:
`zypper install -y fuse3` would be fine.

Also `iptables` needs to be installed.